### ISSUE TITLE
Tests fail with Mojolicious 6.37

### ISSUE THREAD

**Issue by eserte** [zoffixznet/Test-Mojo-Role-SubmitForm#9]
See subject. Statistical analysis shows the following (negative theta is bad):

```
****************************************************************
Regression 'mod:Mojolicious'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    106431226029828080.00
[1='eq_6.22']         0.0000          0.0000       1.36
[2='eq_6.27']         0.0000          0.0000       1.58
[3='eq_6.29']         0.0000          0.0000       0.00
[4='eq_6.31']         0.0000          0.0000       0.00
[5='eq_6.32']         0.0000          0.0000       1.14
[6='eq_6.34']         0.0000          0.0000       4.55
[7='eq_6.35']         0.0000          0.0000       3.41
[8='eq_6.37']        -1.0000          0.0000    -58031600109356968.00

R^2= 1.000, N= 51, K= 9
****************************************************************
```

A failing test output looks like this:

```
#   Failed test 'exact match for JSON Pointer ""'
#   at t/01-tester.t line 28.
Wide character in print at /usr/perl5.16.3t/lib/site_perl/5.16.3/Test/Builder.pm line 1826.
#     Structures begin differing at:
#          $got->{©☺♥} = Does not exist
#     $expected->{©☺♥} = '24'
...
```

**Comment by zoffixznet** [comment]
Thank you. It was been [identified as a problem with Mojolicious](http://irclog.perlgeek.de/mojo/2015-12-16#i_11730215) and once the fix is released, I'll bump the Mojolicious version prereq and release a new version.

**Comment by jhthorsen** [comment]
@zoffixznet: Got any updates on this?

**Comment by zoffixznet** [comment]
Oh, whoops. Totally forgot about this. Yes, it was a problem with a change introduced in Mojolicious 6.37 where instead of using a UTF-8 encoding by default, form params were decoded without encoding or something along those lines.

That was fixed by sri and released in Mojolicious 6.38. My fix for this module involved simply bumping the required Mojolicious version.

**Comment by jhthorsen** [comment]
Thanks!
